Add track descriptions to CFP (#5)
apache/apachecon-eu: content/blog/cfp-open.md

Fortunately, there are mature Big Data Compute projects (like Apache Spark, Apache Hadoop, Apache Flink) and a vibrant ecosystem of related projects within the ASF.

This track welcomes submissions about improving existing projects or creating new ones to solve computing needs in Big Data.
 
**Big Data Storage**:
Big Data is nothing but a massive dataset requiring massive storage capability, and Big Data Storage projects (like Apache HDFS or Apache Ozone) are the backbone for Big Data innovations.
With the continuous growth of data, there's an increasing need for scalable solutions.
Some cloud storage innovations address these scaling requirements, but performance remains a key factor for latency sensitive use cases.
Open source software projects are continually innovating both in cloud and on-premise environments.

This track welcomes submissions related to innovation and improvements in the storage software space.

**Cassandra**:
All things Apache Cassandra, focusing on 5.0 features, what's next in 5.1, how we do engineering, and our community.

**CloudStack**:
Presenting the new Apache CloudStack release, new integrations and capabilities and user stories.

**Community**:

> Community, Governance, and Culture, covering the social and organizational aspects of ASF projects, and the Foundation at large.

Reviewer: +1

> Community, Governance, and Culture, covering the social and organizational aspects of ASF projects, and the Foundation at large.

Reviewer 2: +1

> Community, Governance, and Culture, covering the social and organizational aspects of ASF projects, and the Foundation at large.

Author: ```suggestion
This track covers the Community, Governance, and Culture, including the social and organizational aspects, of ASF projects and the Foundation at large.
```

Performance Engineering in the open-source community is pervasive and includes (but is not limited to) methods and tools for software performance architecting, design, operation, end-user experience, benchmarking, load testing, monitoring, distributed tracing, analysis, prediction, modeling and simulation, regression testing, optimization and tuning, and source code analysis, profiling and instrumentation techniques.

We welcome submissions related to open-source software performance engineering best practices, tools, techniques, results, challenges & lessons learned, etc, particularly demonstrating experiences relevant to:
* Apache software projects
* In-house, managed service and cloud providers of Apache software
* End-users of Apache Software
* The wider open-source, developer and research communities.

**Posters**:
Posters about ASF projects and topics of interest to ASF projects.
Posters with original research are more likely to be accepted.
We are looking for ideas and concepts that are applicable across projects.
Posters do not have to be associated with an ASF project.

> Access/use of all information presented or referenced may not be inhibited by patent or other licensing restrictions. 

Reviewer: +1

> Access/use of all information presented or referenced may not be inhibited by patent or other licensing restrictions. 

Reviewer 2: +1

> Access/use of all information presented or referenced may not be inhibited by patent or other licensing restrictions. 

Author: ```suggestion
Access/use of all information presented or referenced must be unencumbered by by patent or other licensing restrictions. 
```
